Conch/lib/Conch/Time.pm

```diff
@@ -18,49 +18,89 @@ Conch::Time - format Postgres Timestamps as RFC 3337 UTC timestamps
 =head1 METHODS
 
 =cut
+
 package Conch::Time;
 use Mojo::Base -base, -signatures;
+use DateTime::Format::Strptime;
+use Carp 'croak';
+
 use overload
 	'""' => 'to_string',
-	eq => 'compare',
-	ne => sub { !compare(@_) };
-
-use DateTime::Format::Pg;
+	eq   => 'compare',
+	ne   => sub { !compare(@_) };
 
-=head2 datetime
+=head2 timestamp
 
-Underlying L<DateTime> object.
+Underlying RFC 3339 formatted timestamp
 
 =cut
 
-has 'datetime';
+has 'timestamp';
 
 =head2 new
 =cut
+
+my $pg_timestamp_format =
```

Comment:
Perhaps move this into a 'use constant' since it won't change and we certainly want to know if it does?